### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dA505819**
Added geeksforgeeks link for adding packages in anaconda environment to help starters to begin the project with ease.

### CODE REVIEW

**Comment by nileshpatra** [comment]
Hi @dA505819 Thanks for the PR.
SInce the readme follows markdown, can you embed the link in a single word?

For example:
[Here](https://guides.github.com/features/mastering-markdown/) is a tutorial for markdown.
instead of 
https://guides.github.com/features/mastering-markdown/ is a tutorial for markdown.

Also, you can go through the above link to learn as to how to do that, :)

**Comment by dA505819** [comment]
I completely forget about it! Sorry.
Corrected it.

**Comment by nileshpatra** [comment]
@dA505819 what does this mean? 
> "and also help them in this project"

Help whom?
Can you rephrase this?

Also change ' "some" basics' to 'basics' in your sentence.

**Comment by dA505819** [comment]
Updated.

**Comment by nileshpatra** [comment]
LGTM, merging
